tests/integration/test_bundle.py

```diff
@@ -5,16 +5,30 @@
 import inspect
 import logging
 import os
+from os.path import join
 from pathlib import Path
+from urllib.parse import parse_qs, urlparse
 
 import pytest
 import requests
+from lightkube import Client
+from playwright.async_api._generated import Page
 from pytest_operator.plugin import OpsTest
 
+from integration.auth_utils import (
+    auth_code_grant_request,
+    client_credentials_grant_request,
+    construct_authorization_url,
+    userinfo_request,
+)
+from integration.conftest import apply_dex_resources
+
 logger = logging.getLogger(__name__)
 
 TRAEFIK_ADMIN_APP = "traefik-admin"
 TRAEFIK_PUBLIC_APP = "traefik-public"
+DEX_CLIENT_ID = "client_id"
+DEX_CLIENT_SECRET = "client_secret"
 
 
 def get_this_script_dir() -> Path:
```

Comment:
It looks like this function is only used for getting the path for the Jinja template. Is it better if we just replace it with another function to look for the template instead of doing `get_this_script_dir() / ".." / ".." / "bundle.yaml.j2"` multiple times?

Reply:
Good point